### PR TITLE
Migrate to AWS Java SDK v2 

### PR DESCRIPTION
**Pull request by semenodm**

### CODE REVIEW

**Comment by semenodm** [comment]
@ducanhkhu-ovo please review this PR. i'm not sure if you want to discard SDK v1 support

File: src/main/scala/ciris/aws/ssm/package.scala [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
   ): ConfigValue[Param] =
     ConfigValue.resource {
-      Resource {
-        IO {
-          val client =
-            AWSSimpleSystemsManagementClientBuilder
-              .standard()
-              .withRegion(region.asJava)
-              .withCredentials(credentials)
+      Resource
```
Comment (ducanhkhu-ovo) [inline src/main/scala/ciris/aws/ssm/package.scala:22]: is this a typo?

Comment (semenodm) [reply src/main/scala/ciris/aws/ssm/package.scala:22]: sorry don't see it, where exactly ?

Comment (semenodm) [reply src/main/scala/ciris/aws/ssm/package.scala:22]: oh, i see now

**Review by ducanhkhu-ovo — APPROVED** [review]
Thanks for the PR. LGTM 👍

**Comment by semenodm** [comment]
i don't think i can merge it, would be great if you do it, also what is the release process here?

**Comment by ducanhkhu-ovo** [comment]
> i don't think i can merge it, would be great if you do it, also what is the release process here?

Sorry for slow response, I can merge and release. However, I think I oversight the version changes in this PR. Because you switched from internal `Region` to AWS one, it will break compatibility and will be a major version bump instead of just patch

**Comment by semenodm** [comment]
i will completely abide your decision in this question

**Pull request merged**